src/components/Header/styled.ts

```diff
@@ -1,6 +1,21 @@
 import styled from 'styled-components'
+import Toggle from '../Toggle'
+
+export const Wrapper = styled.header`
+	display: grid;
+	grid-template-columns: 1fr 3fr 1fr;
```

Comment:
"The `fr` unit (a “fraction”) is a new unit that can be used when defining grids like any other CSS length such as `%`, `px` or `em`. Tracks sized with `fr` units are called **flexible tracks** as they flex in response to leftover space similar to how flex items with a zero base size fill space in a flex container." 

Hmmm... :thinking: I've never used this one before so I had to do some digging on it. If I'm [reading](https://www.digitalocean.com/community/tutorials/css-css-grid-layout-fr-unit) it correctly, we should always use `fr` for grid sizes? Because it automatically makes the design responsive and to keep consistent? And it's only for grid, in font-size, we still use `em` and probably `%` in anything else?

Reply:
Yes, this is directly related to the CSS grid. It makes elements in a grid proportional to each other (`fr` = fraction). You can read more about grid for example here - https://css-tricks.com/snippets/css/complete-guide-grid/ :+1: